java/flight/flight-sql-jdbc-core/src/test/java/org/apache/arrow/driver/jdbc/client/utils/ClientAuthenticationUtilsTest.java

```diff
@@ -158,7 +158,7 @@ private MockedStatic<KeyStore> createKeyStoreStaticMock() {
           if (method.getName().equals("getInstance")) {
             return invocationOnMock.callRealMethod();
           }
-          return invocationOnMock.getMock();
```

Comment:
If we supply a lambda (`invocationOnMock`), we need to return the method result. Previously, we were just returning the Mock object without invoking the mocked method.